ERROR: https://instagram.com, HTTPSConnectionPool(host='instagram.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x0000018FD6D747D0>, 'Connection to instagram.com timed out. (connect timeout=None)'))
ERROR: https://twitter.com, HTTPSConnectionPool(host='twitter.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x0000018FD6D867D0>, 'Connection to twitter.com timed out. (connect timeout=None)'))
ERROR: https://instagram.com, HTTPSConnectionPool(host='instagram.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001B4E8ED3C90>, 'Connection to instagram.com timed out. (connect timeout=None)'))
ERROR: https://twitter.com, HTTPSConnectionPool(host='twitter.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001B4E8EF6D10>, 'Connection to twitter.com timed out. (connect timeout=None)'))
